website/src/pages/docs/options.mdx

```diff
@@ -166,6 +166,15 @@ Output files into a directory.
 | ----------- | --------------------- | --------------------- |
 | `undefined` | `--out-dir <dirname>` | Only available in CLI |
 
+## index.js template
+
+Specify a template function (API) to change default index.js output (when --out-dir is used).
+
+| Default            | CLI Override          | API Override               |
+| ------------------ | --------------------- | -------------------------- |
+| [`basic template`](https://github.com/gregberge/svgr/blob/master/packages/cli/src/dirCommand.js) | Only available in API | indexTemplate: files => '' |
```

Comment:
It should have the same API as the template option. I mean available in CLI and API.

Reply:
Added --index-template to cli